epoxy-integrationtest/build.gradle

```diff
@@ -48,9 +48,11 @@ dependencies {
   implementation project(':epoxy-annotations')
   implementation project(':epoxy-databinding')
   implementation project(':epoxy-paging')
+  implementation project(':epoxy-viewbinder')
 
   kapt project(':epoxy-processor')
 
+  testImplementation rootProject.deps.androidAppcompat
```

Comment:
could just do
```suggestion
  implementation rootProject.deps.androidAppcompat
```

Reply:
Ah yeah that makes more logical sense, even if it's not used outside tests. I'll change it up but make the commit manually so it's grouped with the other `implementation` declarations